audformat/core/utils.py

```diff
@@ -652,6 +653,37 @@ def join_schemes(
         db.schemes[scheme_id].replace_labels(labels)
 
 
+def map_country(country: str) -> str:
+    r"""Map country to ISO 3166-1.
+
+    Args:
+        country: country string
+
+    Returns:
+        mapped string
+
+    Raises:
+        ValueError: if country is not supported
+
+    Example:
+        >>> map_country('gb')
+        'GBR'
+        >>> map_country('gbr')
+        'GBR'
+        >>> map_country('United Kingdom of Great Britain and Northern Ireland')
+        'GBR'
+
+    """
+    try:
+        result = iso3166.countries.get(country.lower())
+    except KeyError:
+        raise ValueError(
+            f"'{country}' is not supported by ISO 3166."
```

Comment:
```suggestion
            f"'{country}' is not supported by ISO 3166-1."
```

Or does it not matter?